### DISCUSSION TITLE
Wallet running - no listening port error

### DISCUSSION THREAD

**Discussion by shobukle** [Chia-Network/chia-blockchain#6621] [Support]
I got a strange error from `chia wallet show`:
`Exception from 'wallet' "b'\\xce\\x3a\\xc7\\x0a\\xb1ik...`

What does that mean? I stopped the wallet and restarted it but now I just get a connection error:
`Connection error. Check if wallet is running at 9256`

The wallet process is running.

**Comment by loppefaaret** [comment]
You most likely got something stuck or malformed somewhere in the working dir.
I would suggest you try and completely refresh the `.chia\mainnet` folder.
Assuming you are on windows, do the following.
First ensure that everything is installed correctly, you can simply just download and run the 1.1.7 installer.
Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```
Now open up the Chia app again, and readd all your folders that contain plot files, and whatever else kind of settings you had done prior to resetting everything

**Reply by shobukle** [reply]
Thanks. I'm using Linux. Downloading the entire blockchain will take a long time. I'd like to avoid that, if possible.

24 hours later and the wallet is still down.

Aren't there logs or any information to show me what's going on? Everything else seems to be working.

**Reply by loppefaaret** [reply]
turn the \ into / and most of the above works on linux, apart from `chia init`, which either has to be run from the (venv) or from the appropriate folder under linux

there is a `debug.log` under `~/.chia/mainnet/log/`

**Reply by shobukle** [reply]
Unfortunately, there is nothing about the wallet in debug.log (log level: info).

**Comment by shobukle** [comment]
I guess the only solution is to delete the wallet db. That seems ridiculous but I don't see any other solution.

**Reply by loppefaaret** [reply]
try generating a new config with `chia init` first, as i suggested above -  the 2 last commands moves both databases back in

**Comment by BrandtH22** [comment]
@shobukle , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
